test/zero-downtime/README.md

```diff
@@ -0,0 +1,25 @@
+# FlagD Zero downtime test
+
+## How to run
+
+Clone this repository and run the following command to deploy a standalone flagD:
+
+```shell
+IMG=your-flagd-image make deploy-dev-env
+```
+
+This will create a flagd deployment `flagd-dev` namespace.
+
+To run the test, execute:
+
+```shell
+IMG=your-flagd-image IMG_ZD=your-flagd-image2 make run-zd-test
+```
+
+Please be aware, you need to build your two custom images with different tags for flagD first.
```

Comment:
Just for my understanding, is the two image requirement (two tags) come from K8s (ex:- RollingUpdate config) ? Or can this be tests with one image

Reply:
It can be the same image, but needs to have 2 different tags, so that the Rollingupdate of the Deployment is actually executed. If we would use only one image with one tag, no update would be executed (comes from k8s behavior) and therefore the test results are not valid